src/puppetlabs/puppetdb/command.clj

```diff
@@ -387,6 +387,28 @@
         (assoc :payload validated-payload)
         (store-report* start-time db))))
 
+
+;; Configure expiration
+
+{:command "configure expiration"
+ :certname "foo"
+ :version 1
+ :expire {:facts false}}
+
+(defn configure-expiration
+  [{:keys [id received payload]}
+   start-time db]
+  (let [certname (:certname payload)
+        stamp (:producer_timestamp payload (now))
+        expire-facts? (get-in payload [:expire :facts])]
```

Comment:
We'll need behavior similar to how we handle :producer_timestamp with other commands with the :facts_updated timestamp. :facts_updated needs to stay the same across multiple submissions in the sync context so we can make decisions about which "configure expiration" commands are newer than others and need to be synced.